components/kyma-environment-broker/internal/model.go

```diff
@@ -206,16 +206,15 @@ type Operation struct {
 
 	// DEPROVISIONING
 	// Temporary indicates that this deprovisioning operation must not remove the instance
-	Temporary                   bool          `json:"temporary"`
-	ClusterConfigurationDeleted bool          `json:"clusterConfigurationDeleted"`
-	IsServiceInstanceDeleted    bool          `json:"isServiceInstanceDeleted"`
-	Retries                     int           `json:"-"`
-	ReconcilerDeregistrationAt  time.Time     `json:"reconcilerDeregistrationAt"`
-	K8sClient                   client.Client `json:"-"`
+	Temporary                   bool      `json:"temporary"`
+	ClusterConfigurationDeleted bool      `json:"clusterConfigurationDeleted"`
+	Retries                     int       `json:"-"`
+	ReconcilerDeregistrationAt  time.Time `json:"reconcilerDeregistrationAt"`
 
 	// UPDATING
 	UpdatingParameters    UpdatingParametersDTO `json:"updating_parameters"`
 	CheckReconcilerStatus bool                  `json:"check_reconciler_status"`
+	K8sClient             client.Client         `json:"-"`
```

Comment:
this is no longer used in deprovisioning, the only queue that keeps using it appears to be update queue because if I remove it, there are just these build failures

```
env CGO_ENABLED=0 go build -o kyma-environment-broker ./cmd/broker/main.go
# github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process/update
internal/process/update/btp_operator_check.go:33:15: operation.K8sClient undefined (type internal.UpdatingOperation has no field or method K8sClient)
internal/process/update/btp_operator_check.go:37:63: operation.K8sClient undefined (type internal.UpdatingOperation has no field or method K8sClient)
internal/process/update/get_kubeconfig_step.go:44:13: operation.K8sClient undefined (type internal.UpdatingOperation has no field or method K8sClient)
internal/process/update/get_kubeconfig_step.go:75:12: operation.K8sClient undefined (type internal.UpdatingOperation has no field or method K8sClient)
```